style.css

```diff
@@ -0,0 +1,483 @@
+}
+
+.box3:hover .overlay {
+  opacity: 0.6;
+  line-height: 2rem;
+}
+
+.box4:hover .overlay {
+  opacity: 0.6;
+  line-height: 2rem;
+}
+
+.box5:hover .overlay {
+  opacity: 0.6;
+  line-height: 2rem;
+}
+
+
+/* content within the overlays on boxes */
+.overlay_contents {
+  margin: auto;
+  width: 80%;
+  text-align: center;
+}
+
+.text {
+  margin: 100px 0;
+}
+
+.portfolio_link {
+  padding: 0.5rem;
+  border: 1px solid #000000;
+}
+
+
+
+
+/* separate portfolio page */
+
+#mayden_logo {
+  padding-top: 120px;
+}
+
+#pilot_shop {
+  padding-top: 120px;
+}
+
+.portfolio_main_image {
+  width: 100%;
+  height: 550px;
+  background-color: #f2f2f2;
+}
+
+.project_section {
+  margin-bottom: 120px;
+}
+
+
+
+/* contact section */
+.contact_panel {
+  height: 320px;
+  /*background-color: #6BC4BA;*/
+  background-color: #dbdbdb;
+  padding-top: 40px;
+}
+
+.center_text {
+	text-align: center;
+}
+
+.social_media_box {
+
+	display: inline-block;
+}
+
+.social_media_box a {
+	margin: 0;
+}
+
+.social_media_box img {
+	max-width: 80px;
+	padding: 20px 30px;
+}
+
+.contact_panel_a {
+	display: block;
+}
+
+.email {
+	border-bottom: 1px solid #dbdbdb;
+	padding-bottom: 10px;
+	margin-top: 18px;
+}
+
+/* to add underline, when you hover over the email: */
+.email:hover {
+	border-bottom: 1px solid #000000;
+	padding-bottom: 10px;
+}
+
+
+
+
+/* footer section */
+footer {
+  background-color: #f7d320;
+  height: 50px;
+  overflow: auto;
+}
+
+.copyright {
+  font-weight: 200;
+  font-style: italic;
+  font-family: 'Futura', 'Helvetica', sans-serif;
+  font-size: 0.8rem;
+  float: left;
+  margin: 11px auto;
+}
+
+.move_to {
+  float: right;
+  margin: 16px auto;
+  font-weight: 700;
+}
+
+
+/* media queries */
+
+@media only screen and (max-width: 770px) {
+  /* mobile size*/
+
+  /* nav bar section*/
+  nav {
+    height: 90px;
+  }
+
+  .top_nav_content a {
+    float: none;
+    width: auto;
+    display: block;
+    margin: 10px 0 0 0;
+    font-size: 0.9rem;
+  }
+
+  .my_name {
+    width: 150px;
+    height: 75px;
+  }
+
+  .logo_hannah,
+  .logo_golightly,
+  .logo_dot {
+    font-size: 1.5rem;
+  }
+
+  /* yellow callout section*/
+  /* 'hey there' text */
+  .callout h2 {
+    font-size: 3rem;
+    line-height: 5rem;
+  }
+
+  .callout_right {
+    margin: 5% 0 0 10%;
+    display: block;
+  }
+
+  h2 .callout_right {
+    text-align: right;
+  }
+
+  header {
+    height: 490px;
+    padding: 30% 0 20% 0;
+  }
+
+  /* about me section*/
+
+  .about_me,
+  .img_me_box {
+    width: 100%;
+  }
+
+  .img_me {
+    height: 400px;
+  }
+
+  /* portfolio section*/
+
+  .box1,
+  .box2,
+  .box3,
+  .box4,
+  .box5 {
+    width: 100%;
+    height: 300px;
+  }
+  
+
+  /* images in portfolio page */
+  .portfolio_main_image {
+    height: 400px;
+  }
+
+
+/* contact section */
+  
+.social_media_box img {
+	max-width: 60px;
+	padding: 10px 20px;
+}
+
+.email {
+	font-size: 0.9rem;
+	margin: 20px 2px;
+}
+
+.contact_panel {
+	height: 280px;
```

Comment:
Watch your indents here, these are inside a media query so should be indented like the rest

Reply:
amended